Source_Code/TM_Backend/TeamMentor.Users/Users/Users_Creation.cs

```diff
@@ -239,6 +251,24 @@ public static Signup_Result createTmUserResponse(this TM_UserData userData, NewU
         }
 
         #region Sigup validations, shorter and specialized methods
+        private static Signup_Result ValidateEmailPattern(TMConfig config)

```

Comment:
this should be an extension method and the name is wrong (since this does not validate an email, just creates the error message to send to the user)